read.pl

```diff
@@ -125,19 +125,21 @@ sub inner_read_report {
       $i++;
       next;
     }
-    if ( $_ =~ /^\s*(\d+) / ) {
+    if ( $_ =~ /^\s{0,5}(\d+) / ) {
```

Comment:
The proper way to fix this, I think, would be to force ``task`` to show each task on a single line (by setting ``defaultwidth`` to 0), and then handling our own line breaking and layout.  Of course that would be quite a departure from vit's current model, in which it simply shows whatever ``task`` outputs, so I'm not entirely sure if that is the way to go.

Alternatively, using a regexp here might be the wrong way to look at it.  What we really should be checking is if the first column is empty (in which case the line is a continuation of the previous task), or if is isn't (in which case this is a new task).  That shouldn't be too hard to implement.